### ISSUE TITLE
webapps.uni-koeln.de/tamil/

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/Cologne#65]
webapps.uni-koeln.de/tamil/ is the first page to be found if I search for `cologne sanskrit` and http://www.sanskrit-lexicon.uni-koeln.de/scans/MWScan/tamil/index.html links to http://www.sanskrit-lexicon.uni-koeln.de/monier/ (2008 revision) - better not. Let's change to http://www.sanskrit-lexicon.uni-koeln.de/scans/MWScan/2014/web/index.php ?

**Comment by funderburkjim** [comment]
Done -
  Bet you thought I forgot!

This came to my attention recently, since Cologne is changing the server for sanskrit-lexicon.

**Comment by gasyoun** [comment]
`since Cologne is changing the server for sanskrit-lexicon` - could you please share more details?

**Comment by funderburkjim** [comment]
@gasyoun All I know is that the webmaster at Cologne said that the server used to host the sanskrit-lexicon project was being changed -  I think he mentioned the new server would be using RHEL6, while the former server used RHEL5.  I've done some testing and changed a few things for compatibility, using a temporary url.  All the problems I noticed have been fixed, and   today the webmaster emailed me that the new server is now the one hosting sanskrit-lexicon.  Hopefully I caught all the compatibility issues, and users won't be adversely affected.  I also hope that response time, etc. will continue to be good on the new server.
